### ISSUE TITLE
Texture issue

### ISSUE THREAD

**Issue by ambrosiogabe** [Shadowblitz16/Jade#1]
Hey ShadowBlitz, this is GamesWithGabe!

I took a look at your code and it looks like this line is the reason you're having problems, you have a plus instead of a multiplication symbol. 

https://github.com/Shadowblitz16/Jade/blob/34e07fee270a97e8746853c8f573f617010b85ec/src/main/java/jade/LevelEditorScene.java#L108

So if you change this line to:
```java
glVertexAttribPointer(2, uvSize, GL_FLOAT, false, vertexSizeBytes, (positionSize + colorSize) * Float.BYTES); // Notice I did '*' Float.BYTES
```
you should be good. I noticed a couple other things though, I'm not sure what system you're using, but I'm on Windows and I ran into a problem with your shader Java code. On windows the `System.lineSeparator()` is a '\r\n' character, and that makes the `indexOf()` function fail since it's two characters big. If you change your shader code to use '\n' it should work on all platforms, that was at this line of your code:

https://github.com/Shadowblitz16/Jade/blob/34e07fee270a97e8746853c8f573f617010b85ec/src/main/java/jade/renderer/Shader.java#L35

and it was here:

https://github.com/Shadowblitz16/Jade/blob/34e07fee270a97e8746853c8f573f617010b85ec/src/main/java/jade/renderer/Shader.java#L40

If you change those to '\n' instead of `System.lineSeparator()` it should be cross-platform :)

Last thing that I'll mention is that your texture was blurry. If you want it to look pixelated you can change this line:

https://github.com/Shadowblitz16/Jade/blob/34e07fee270a97e8746853c8f573f617010b85ec/src/main/java/jade/renderer/Texture.java#L28

and this line:

https://github.com/Shadowblitz16/Jade/blob/34e07fee270a97e8746853c8f573f617010b85ec/src/main/java/jade/renderer/Texture.java#L30

to use GL_NEAREST, which should give it a pixelated look :)

**Comment by Shadowblitz16** [comment]
@ambrosiogabe thankyou very much for the help
The multiplications and plus is hard to tell then difference when you have 1 small monitor and you split youtube and the ide across the screen. thankyou.

The line separator thing was me having trouble with the way you did it so I just googled how to do cross platform line seperator, I guess the person that suggested that was wrong.

also I was using GL_NEAREST by I read somewhere that it could cause a black texture so I tried it with GL_LINEAR to try to fix the issue, and didn't have time to change it back because I had to goto sleep

anyways thankyou your tutorials are one of the most friendly I have come across.
